wordpress-mobile/WordPress-iOS
# HACK week: Add new tab icons

## Description
- Adds new tab icons (Design ref: eYeHXEMDbnFptE40xUqZ2T-fi-1862%3A3175)
- Controlled by a local feature flag "New Tab Icons"

## How to test
- Verify new tab icons adapt to environment (light vs dark)
- Verify notification icon shows the correct badge color when there are unread notifications (blue for WP, green for JP)
- Verify you can toggle the "New Tab Icons" feature flag

https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/b1a64a65-2216-4468-8ce2-dff30bea8248



## Regression Notes
1. Potential unintended areas of impact
n/a

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

3. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
